.github/workflows/main.yml

```diff
@@ -9,6 +9,38 @@ on:
   workflow_dispatch:
 
 jobs:
+  lint:
+    name: "Run linters"
+    runs-on: "ubuntu-latest"
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+      prerelease: ${{ steps.version.outputs.prerelease }}
+    steps:
+      - uses: "actions/checkout@v2"
+      - uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+
+      - name: "Install poetry, check-wheel-content, and twine"
```

Comment:
The Poetry part in the description no longer seems to be valid. Could you adjust it?